src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * A custom checked exception class to represent
+ * exceptions specific to Duke
+ */
```

Comment:
Perhaps it may be better to put the class level JavaDoc description above the class declaration statement instead of at the top of the file?